File: src/main/java/me/yellowbear/uwujobs/jobs/BlockPlace.java
package me.yellowbear.uwujobs.jobs;

public enum BlockPlace {
    BUILDER;
    public static BlockBreak getJob(String job) {
        for (BlockBreak jobs : BlockBreak.values()) {
            if (jobs.name().equalsIgnoreCase(job)) {
                return jobs;
            }
        }
        return null;
    }
}

File: src/main/java/me/yellowbear/uwujobs/jobs/BlockPlace.kt
package me.yellowbear.uwujobs.jobs

enum class BlockPlace {
    BUILDER;

    companion object {
        fun getJob(job: String?): BlockBreak? {
            for (jobs in BlockBreak.entries) {
                if (jobs.name.equals(job, ignoreCase = true)) {
                    return jobs
                }
            }
            return null
        }
    }
}
